### PR TITLE
Log offset deviations

### PR DESCRIPTION
**Pull request by simonwahlgren**

### CODE REVIEW

File: eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py [hunk 1]
```diff
@@ -67,7 +69,17 @@ def seen(self, message: Message) -> bool:
         last_offset = self.redis.get(self._key(message))
         if last_offset is None:
             return False
-        return message._meta.offset <= int(last_offset)
+
+        # https://github.com/edenhill/librdkafka/issues/1720
+        # until this bug is fixed at least make sure we log when it happens.
+        offset = message._meta.offset
+        offset_diff = offset - last_offset
```
Comment (saabeilin) [inline eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py]: `last_offset` should be coerced to `int` first I guess :)

File: eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py [hunk 2]
```diff
@@ -67,7 +69,17 @@ def seen(self, message: Message) -> bool:
...
+        if offset_diff > 1:
```
Comment (saabeilin) [inline eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py]: Shall we also log when it becomes negative? ;)

File: eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py (hunk 1 above)
Comment (simonwahlgren) [reply eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py]: Thanks! :)

File: eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py (hunk 2 above)
Comment (simonwahlgren) [reply eventsourcing_helpers/messagebus/backends/kafka/offset_watchdog/backends/redis/__init__.py]: I think that's a good idea! I will add it

**Pull request merged**